internal/scribe/src/samsara/scribe/serializer/transit.clj

```diff
@@ -0,0 +1,31 @@
+(ns samsara.scribe.serializer.transit
+  (:refer-clojure :exclude [read])
+  (:import [java.io ByteArrayInputStream ByteArrayOutputStream])
+  (:require [cognitect.transit :as transit]
+            [samsara.scribe.protocol :refer :all]))
+
+
+(def ^:const DEFAULT-CONFIG
+  {:size 1024, :format :json})
+
+
+(defrecord TransitScribe [config]
+
+  Scribe
+
+  (write [_ data]
+    (let [out (ByteArrayOutputStream. (config :size))
+          writer (transit/writer out (config :format))]
+      (do (transit/write writer data) out)))
+
+  (read [_ bytez]
+    (let [in (ByteArrayInputStream. (.toByteArray bytez))
```

Comment:
`.toByteArray` should be probably added to the `write` function like `(.toByteArray out)`

anyway run `lein check` to ensure no reflection is used for these calls.